app/assets/javascripts/admin/task/task_create_form_view.js

```diff
@@ -311,27 +313,27 @@ class TaskCreateFormView extends React.PureComponent<Props, State> {
               </FormItem>
 
               {this.state.isNMLSpecification ? (
-                <FormItem label="NML File" hasFeedback>
-                  {getFieldDecorator("nmlFile", {
+                <FormItem label="NML Files" hasFeedback>
+                  {getFieldDecorator("nmlFiles", {
                     rules: [{ required: true }],
                     valuePropName: "fileList",
                     getValueFromEvent: this.normFile,
                   })(
                     <Upload.Dragger
                       accept=".nml,.zip"
-                      name="nmlFile"
+                      name="nmlFiles"
                       beforeUpload={file => {
-                        this.props.form.setFieldsValue({ nmlFile: [file] });
+                        this.props.form.setFieldsValue({ nmlFiles: [file] });
```

Comment:
@philippotto `beforeUpload` is called for every file that is dragged onto the upload section of the form. I would have suspected that the forms fieldValue `nmlFiles` would be overwritten by subsequent "drags" of files, but it is not, although `setFieldsValue` is called repeatedly with different files. Do you know why this works?

Reply:
Hmm, seems definitely wrong. The antd documentation lists an example like this:

```
      beforeUpload: (file) => {
        this.setState(({ fileList }) => ({
          fileList: [...fileList, file],
        }));
        return false;
      },
```

However, this sets the state (as that is needed for some special rendering). 

My assumption is that we can remove the entire `beforeUpload` hook, since antd manages this by itself, anyway? And that's why it works at the moment?